rpctest/node.go

```diff
@@ -167,9 +163,11 @@ func (n *nodeConfig) String() string {
 
 // cleanup removes the tmp data and log directories.
 func (n *nodeConfig) cleanup() error {
+	logDirParent := strings.Replace(n.logDir, "/logs", "", 1)
```

Comment:
These will be the same directory.

This whole function can just be changed to:

```Go
	return os.RemoveAll(filepath.Dir(n.DataDir))
```